### ISSUE TITLE
media-no-missing-colon

### ISSUE THREAD

**Issue by stephenway** [stylelint/stylelint#692]
I was surprised by a simple omission of the `:` character in some of my media queries.

```css
@media only screen and (max-width 34em) { ... }
```

I don't see a rule that checks for this, should this be a feature suggestion for something like `media-no-missing-colon`?

**Comment by davidtheclark** [comment]
Seems like a great rule to me.

**Comment by davidtheclark** [comment]
Thinking a little about the scope and name of this rule.

- Should name it `media-feature-*` to match the established convention.
- It needs to allow range operators as well as colons. So the name needs a word that identifies both. I'm struggling to find one. `media-feature-no-missing-punctuation`?
